app/src/main/java/com/cornellappdev/android/eatery/MenuActivity.java

```diff
@@ -223,7 +249,9 @@ public ViewPagerAdapter(Context context, FragmentManager manager) {
     public void setPrimaryItem(ViewGroup container, int position, Object object) {
       super.setPrimaryItem(container, position, object);
       if (position != mCurrentPosition) {
-        if (mCurrentPosition == -1) position = 0;
+        if (mCurrentPosition == -1) {
+          position = 0;
+        }
```

Comment:
Here I think leaving the original snippet is fine also!

Reply:
This violates the Google style guide.

Reply:
https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used